conf/mod-zone-difficulty.conf

```diff
@@ -0,0 +1,78 @@
+#
```

Comment:
There can not be a .conf file. We only distribute `.conf.dist` files.

Reply:
delete conf and added to ignore